### PR TITLE
Update s3secrets version, and pull artifactory creds

### PR DESCRIPTION
**Pull request by jaykeshur**

### CODE REVIEW

**Comment by vaijab** [comment]
s3secrets syntax is actually right, because:
https://github.com/UKHomeOffice/jenkins-docker-aws/blob/master/Dockerfile#L39

That's the original s3secrets I wrote! :-) If you want the new syntax, then you need to also update s3secrets version too.

**Comment by jaykeshur** [comment]
updated to the latest version

**Comment by vaijab** [comment]
> updated to the latest version

@jaykeshur if you could also change PR title to match what you're actually changing now.

File: jenkins.sh [hunk 1]
```diff
@@ -35,19 +35,19 @@ set_kubeconfig() {
   echo "Kubeconfig created successfully"
 }
 
-set_quay_login() {
-  echo "Creating docker login for quay.io"
+set_artifactory_login() {
```
Comment (jon-shanks) [inline jenkins.sh]: Shouldn't this just be set_docker_login() { .. so it is kept generic?

File: jenkins.sh [hunk 2]
```diff
@@ -35,19 +35,19 @@ set_kubeconfig() {
...
+  echo "Docker login for artifactory created successfully"
 }
 
 # Allow to pass in jenkins options after --
 if [[ $# -lt 1 ]] || [[ "$1" == "--"* ]]; then
   if [ -n "$SECRETS_BUCKET" ]; then
     download_secrets
     set_kubeconfig
-    set_quay_login
+    set_artifactory_login
```
Comment (jon-shanks) [inline jenkins.sh]: and here too

File: jenkins.sh [hunk 3]
```diff
@@ -35,19 +35,19 @@ set_kubeconfig() {
...
 
-set_quay_login() {
-  echo "Creating docker login for quay.io"
+set_docker_login() {
+  echo "Creating docker login"
   mkdir /root/.docker
-  cp /root/.secrets/dockercfg /root/.docker/config.json
-  echo "Docker login for quay.io created successfully"
+  cp /root/.secrets/config.json /root/.docker/config.json
+  echo "Docker login created successfully"
```
Comment (vaijab) [inline jenkins.sh:42]: Before saying that, you could check whether it really has been successfully created.

**Comment by vaijab** [comment]
Otherwise LGTM

**Pull request merged**